app/controllers/api/issues_controller.rb

```diff
@@ -25,6 +25,14 @@ def update_issue
     def label_issue
```

Comment:
I don't think we can do label updates without an array of all the labels, otherwise GitHub just erases all the omitted labels from the array. All the HuBoards stay up to date but the actual GH data doesn't change.